composer.json

```diff
@@ -21,12 +21,12 @@
         }
     ],
     "require": {
-        "php": "^7.2"
+        "php": "^7.2 || ^8.0",
+        "symfony/phpunit-bridge": "^5.1.7"
```

Comment:
That looks like a mistake or why did you move that from `require-dev` to `require`?

Reply:
👍